SQL/0000-00-01-Permission.sql

```diff
@@ -109,7 +109,9 @@ INSERT INTO `permissions` VALUES
     (45,'issue_tracker_reporter', 'Can add a new issue, edit own issue, comment on all', 2),
     (46,'issue_tracker_developer', 'Can re-assign issues, mark issues as closed, comment on all, edit issues.', 2),
     (47,'imaging_browser_phantom_allsites', 'Can access only phantom data from all sites in Imaging Browser', 2),
-    (48,'imaging_browser_phantom_ownsite', 'Can access only phantom data from own site in Imaging Browser', 2);
+    (48,'imaging_browser_phantom_ownsite', 'Can access only phantom data from own site in Imaging Browser', 2),
+    (49,'data_release_upload', 'Data Release: Upload file', 2),
+    (50,'data_release_edit_permissions', 'Data Release: Give user permission to view files', 2);
```

Comment:
This also gives a user permissions to edit files, not just view

Reply:
no... again this is confusing. what this permission does is it allows the user that has it to GIVE ANOTHER user viewing permission for files (no edit) i.e. if user A has "edit" permission, user A can then EDIT the viewing permissions of user B on a file..


@johnsaigle if you get what I mean maybe you can formulate it better. As I suggested for the permission code, maybe replace with "Give user access to file" ?

Reply:
How about something like "Grant other users view-file permissions"?

Reply:
sounds good
what about the code field ?